### ISSUE TITLE
EPPlus doesn't support all excel number formats

### ISSUE THREAD

**Issue by rafaeltab** [EPPlusSoftware/EPPlus#344]
Hi there,

I was creating a data field in epplus and wanted to set the language of the cell using number format. 
In excel this would have the following number format for the dutch language: `[$-nl]d mmmm yyyy` and this would display as `29 maart 2021`
Or for english: `[$-en]d mmmm yyyy` which would display as `29 March 2021`

However, this does not seem to be supported by epplus as it throws the following error: 
```
 System.FormatException: Input string was not in a correct format.
  at System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
  at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
  at OfficeOpenXml.Style.XmlAccess.ExcelNumberFormatXml.ExcelFormatTranslator.ToNetFormat(String ExcelFormat, Boolean forColWidth)
  at OfficeOpenXml.Style.XmlAccess.ExcelNumberFormatXml.ExcelFormatTranslator..ctor(String format, Int32 numFmtID)
  at OfficeOpenXml.Style.XmlAccess.ExcelNumberFormatXml.get_FormatTranslator()
  at OfficeOpenXml.ExcelRangeBase.GetFormattedText(Boolean forWidthCalc)
  at OfficeOpenXml.ExcelRangeBase.AutoFitColumns(Double MinimumWidth, Double MaximumWidth)
```

**Comment by JanKallman** [comment]
This is a bug when using localized formats. Fixed in latest develop, so give it a try using our develop Nuget feed... https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

**Comment by JanKallman** [comment]
Fixed in version 5.6.2
